templates/vue/src/components/Lightbox/media/VideoMedia/UrlVideoMedia.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div :class="['video-container', { fullscreen: node.fullscreen }]">
+    <video
+      ref="video"
+      controls
+      :src="node.typeData.mediaURL"
+      :style="videoStyles"
+      @loadeddata="handleLoad"
+      @play="handlePlay"
+      @pause="handlePause"
+      @seeked="handleSeek"
+      @seeking="$emit('seeking')"
+      @timeupdate="updateVideoProgress"
+    ></video>
+  </div>
+</template>
+
+<script>
+import client from "@/services/TapestryAPI"
```

Comment:
Can we add a comment to explain what type of media this is? (e.g., how is this different from YoutubeMedia?)

Reply:
Seems like from `VideoMedia` this is the uploaded videos?